webapp/src/js/controllers/inbox.js

```diff
@@ -177,7 +177,7 @@ var feedback = require('../modules/feedback'),
       delete $window.startupTimes;
     });
 
-    feedback.init({
+    Feedback.init({
```

Comment:
Now that you're injecting DB and Session we don't need to pass these functions in as options any more.

You can even inject `APP_CONFIG` then you don't even need to have a `Feedback.init` function any more!

Reply:
Great suggestion! I kept the init function because it is also used to submit bootstrap errors and intercept console logging. And it is easier to test with init function (mock console and window)